### DISCUSSION TITLE
Server Error while github repository create

### DISCUSSION THREAD

**Discussion by zakariaaitoumeriem1889** [community/community#22159] [New to GitHub]
<p>Hello Everyone,<br>
When I create my github repository, a error server alert display.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/0/40d456d6a0ea7f0622680cdccb41bc5a29a4d51b.jpeg" data-download-href="/uploads/short-url/9fvuydGdGKlLKKHOFEDG7x9lMo3.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/45511250/181069270-2eb95255-5add-4d00-9e31-960757f45def.jpeg" alt="image" data-base62-sha1="9fvuydGdGKlLKKHOFEDG7x9lMo3" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40d456d6a0ea7f0622680cdccb41bc5a29a4d51b_2_690x388.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40d456d6a0ea7f0622680cdccb41bc5a29a4d51b_2_1035x582.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40d456d6a0ea7f0622680cdccb41bc5a29a4d51b_2_1380x776.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40d456d6a0ea7f0622680cdccb41bc5a29a4d51b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 307 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
Please help me for fixing this problems.

**Comment by ernest-phillips** [comment]
<p>Hi <a class="mention" href="/zakariaaitoumeriem1889">@zakariaaitoumeriem1889</a> :wave:</p>
<p>In order for the community to best assist you, please list all of the steps you took to get to this point.</p>
<p>This way, we can attempt to reproduce the error.</p>

**Comment by zakariaaitoumeriem1889** [comment]
<p>Hi everyone,<br>
Firstly, I had a directory with the same name which I deleted. After, I clicked on new repository. I created a new repo with the same name and when I clicked on Create repository the error message popped up.</p>

**Comment by canuckjacq** [comment]
<p>If you did that all in rapid succession it could be that the repository still existed when you were creating the new one and it may have glitched.</p>
<p>I can see there is one repository on your account that did encounter something during creation - so it’s not exactly usable. Do you want me to try and delete it for you, so you can try to create it again?</p>

**Comment by zakariaaitoumeriem1889** [comment]
<p>Hello. You can delete it. Thank you.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>I’ve started that now. Please give it an hour or so before trying to create a new one with the same name, and hopefully that will work!</p>
